src/screens/feedback-screen/index.js

```diff
@@ -91,6 +107,64 @@ const FeedbackScreen = () => {
       comment: '',
       userRole: role,
     };
+    const {
+      authToken,
+      confname,
+      metadata = {},
+    } = meetingData.current;
+
+    const getDeviceType = () => {
+      if (Platform.OS === 'ios') {
+        return Platform.constants.interfaceIdiom;
+      }
+      return Platform.constants.uiMode;
+    };
+
+    const feedback = {
+      timestamp: new Date().toISOString(),
+      rating,
+      session: {
+        session_name: confname,
+        institution_name: metadata.name,
+        institution_guid: metadata[CUSTOMER_METADATA.guid],
+        session_id: payload.meetingId,
+      },
+      device: {
+        type: getDeviceType(),
+        os: Platform.OS,
+        browser: APP_IDENTIFIER,
+        nativeAppDeviceInformation: {
+          brand: DEVICE_INFORMATION.brand,
+          designName: DEVICE_INFORMATION.designName,
+          name: DEVICE_INFORMATION.deviceName,
+          yearClass: DEVICE_INFORMATION.deviceYearClass,
+          manufacturer: DEVICE_INFORMATION.manufacturer,
+          modelId: DEVICE_INFORMATION.modelId,
+          modelName: DEVICE_INFORMATION.modelName,
+          osBuildId: DEVICE_INFORMATION.osBuildId,
+          osInternalBuildId: DEVICE_INFORMATION.osInternalBuildId,
+          osName: DEVICE_INFORMATION.osName,
+          osVersion: DEVICE_INFORMATION.osVersion,
+          platformApiLevel: DEVICE_INFORMATION.platformApiLevel,
+          productName: DEVICE_INFORMATION.productName,
+          suppCpuArch: DEVICE_INFORMATION.supportedCpuArchitectures,
+          totalMemory: DEVICE_INFORMATION.totalMemory
+        },
```

Comment:
Wouldn't it be better to change the name of the object from DEVICE_INFORMATION
to nativeAppDeviceInformation and link it directly here? Or just destructure the 
DEVICE_INFORMATION since the variables are the same?